reactor-core/src/main/java/reactor/core/scheduler/Scheduler.java

```diff
@@ -35,7 +36,7 @@
  * @author Stephane Maldini
  * @author Simon Baslé
  */
-public interface Scheduler extends Disposable {
+public interface Scheduler extends Disposable, Scannable {
```

Comment:
Like Flux we don't need to expose Scannable in the surface API, as Scannable.from(Object) should introspect already without polluting the surface.